model-optimizer/extensions/front/tf/CTCGreedyDecoder_ext.py

```diff
@@ -24,7 +24,7 @@ class CTCCGreedyDecoderFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         attrs = {
-            'ctc_merge_repeated': int(node.pb.attr['merge_repeated'].b),
+            'merge_repeated': bool(node.pb.attr['merge_repeated'].b),
```

Comment:
Did you verify that if you read IR generated in previous release with CTCGreedyDecoder using the MO IR Reader it is correctly read and saved?

Reply:
Checked both models.  IR Reader works correct.